web/book/src/tutorial/relations.md

```diff
@@ -1,78 +1,110 @@
 # Relations
 
-PRQL is designed on top of relational algebra, which is the an established data
-model used by modern databases. _Relation_ has a rigid mathematical definition,
-which can be simplified down to "tabular data". See table of `albums` for
-example:
-
-| album_id | title               | artist_id |
-| -------- | ------------------- | --------- |
-| 1        | For Those About ... | 1         |
-| 2        | Balls to the Wall   | 2         |
-| 3        | Restless and Wild   | 2         |
-| 4        | Let There Be Rock   | 1         |
-| 5        | Big Ones            | 3         |
-| 6        | Jagged Little Pill  | 4         |
-
-It is composed of columns, each of which has an unique name and a designated
-data type. `album_id` has a data type of "integer numbers" and `title` has data
-type of "text".
-
-> Side note: most definitions use unordered relations, which cannot have
-> duplicate rows. PRQL defines relations to have an order and to be able to
-> contain duplicate rows.
-
-PRQL is designed to query relations such as the `albums` table above. The most
-basic query would be:
+PRQL is designed on top of _relational algebra_, which is the established data
+model used by modern SQL databases.
+A _relation_ has a rigid mathematical definition,
+which can be simplified to "a table of data".
+For example, the `invoices` table from the Chinook database[^1] looks like this:
+
+| invoice_id | customer_id | billing_city | _other columns_ | total |
+| ---------- | ------------ | ------------ | :-----------: | ----- |
+| 1        |  2 | Stuttgart | ...  | 1.98 |
+| 2        |  4 | Oslo      | ...        | 3.96 |
+| 3        |  8 | Brussels  | ...        | 5.94 |
+| 4        | 14 | Edmonton  | ...         | 8.91 |
+| 5        | 23 | Boston    | ...         | 13.86 |
+| 6        | 37 | Frankfurt | ...         | 0.99 |
+
+A table (relation) is composed of columns, each of which has an unique name and a designated data type.
+Every table has zero to many rows, each containing the same set of columns.
+The table above has `invoice_id` and `customer_id` columns with a data type of "integer number",
+a `billing_city` column with a data type of "text",
+a number of other columns, and
+and a `total` column that contains floating-point number. [^2]
+
+## Queries
+
+PRQL is designed to build queries that combine and select data from relations such as the `invoices` table above. Here is the most basic query:
 
 ```
-from albums
+from invoices
 ```
 
-This is not impressive, it is just the same table as we had before. But we can
-add on to it:
+_Note: You can try each of these examples here in the [Playground.](https://prql-lang.org/playground/)
+Enter the query on the left-hand side,
+and click **output.arrow** in the right-hand side to see the result._
+
+The result of the query above is not terribly interesting, it's just the same table as before.
+But PRQL can do more...
+
```

Comment:
```suggestion
```

Reply:
I wanted to motivate the notions that a) "just type it in the Playground" and b) a bare `from ....` transform generates a sensible, human-readable result.